### DISCUSSION TITLE
Add User Management

### DISCUSSION THREAD

**Discussion by drupaladmin** [Releem/releem-agent#279] [Feature Requests]
A basic feature that your saas doesn't have is the User Management. In a business, there always a guy who pay and a guy how used it, and not we need to shared password.

**Comment by p10ch** [comment]
As an MSP who is typically responsible for the overall admin and billing, my customers need access to audit and may want me to delegate access to dba/dev to apply. The simplest I envisage would be for the account owner of multiple instances, to be able to create and revoke tokens that grant either read-only or read+apply permissions to specific instances. These tokens can then be managed and distributed by the owner as needed.

**Reply by drupaladmin** [reply]
@p10ch , thank you for sharing detailed workflow

**Comment by drupaladmin** [comment]
Released in [1.22.1](https://github.com/Releem/mysqlconfigurer/releases/tag/1.22.1)

Documentation published here: https://docs.releem.com/server-settings/invite-users-and-assign-roles
